source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -44,104 +112,112 @@ When creating placeholder text, use the following guidelines.
 
    * - Within regular text, show placeholder text in italics.
 
-       Within code samples, use the RST ``:samp:`` directive, and
-       enclose the placeholder text in curly braces. This formatting
-       renders the placeholder in italics.
+       Use the
+         ``.. code-block:: <lexer>`` for block code examples.
 
        If you can't apply text formatting to the code, enclose
        placeholders in punctuation that doesn't have any other special
-       use in the code. For example, use angle brackets or curly
-       braces. Use a consistent convention throughout the documentation
-       set.
+       use in the code. Use a consistent convention throughout the documentation set per the programming language.
+
+     - .. code-block:: console
 
-     - :samp:`nova boot {serverName} --image {image} --flavor {flavor}
-       --nic net-id=net1_id`
+          mongod --port <portNumber> \
+                 --dbpath <tempDatabasePath> \
+                 --setParameter ttlMonitorEnabled=false
 
    * - Use lowercase letters except when showing a multiple-word
        placeholder.
 
        To show a multiple-word placeholder, don't separate the words
        with spaces or symbols. To distinguish the words in the
        placeholder, capitalize the first letter of each word after the
-       first word (called camelCase). Don't capitalize the first word.
+       first word (called :wikipedia:`camelCase </Camel_case>`). Never
+       capitalize the first word.
```

Comment:
Nitpick: this probably should mention the case in which it may start with a capital letter. Perhaps "Never change the first letter from lowercase to uppercase".